clang/lib/Driver/Driver.cpp

```diff
@@ -608,6 +608,20 @@ Driver::OpenMPRuntimeKind Driver::getOpenMPRuntime(const ArgList &Args) const {
   return RT;
 }
 
+static bool isValidSYCLTriple(llvm::Triple T) {
+  // Check for invalid SYCL device triple values.
+  StringRef A(T.getArchName());
```

Comment:
Minor: suggest moving the initialization closer to its usage

Reply:
Done